src/YoutubeAtom.tsx

```diff
@@ -39,6 +42,7 @@ type Props = {
     origin?: string;
     eventEmitters: ((event: VideoEventKey) => void)[];
     pillar: ArticleTheme;
+    isMainMedia: boolean;
```

Comment:
Can this be an optional prop?

Reply:
It could. But isn't it better for props to be required? As in, if I _know_ I have the prop I can make decisions based on it, but if it's required I need to guard against when it's missing and make assumptions